faasmcli/faasmcli/tasks/k8s.py

```diff
@@ -1,8 +1,8 @@
-from time import sleep
+from datetime import datetime
```

Comment:
Very messy diff. You may check the raw file [here](https://github.com/faasm/faasm/blob/remove-knative/faasmcli/faasmcli/tasks/k8s.py).